ui/src/app/shared/components/input-filter.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {Autocomplete} from '../../../../node_modules/argo-ui';
+import {ErrorPanel} from './error-panel';
+
+interface InputProps {
+    value: string;
+    placeholder: string;
+    type: string;
```

Comment:
what is "type"? is in `select` or `checkbox`?

Reply:
It is used to generate the local storage key. I renamed it to `name`.